includes/class-wcsatt-display.php

```diff
@@ -526,6 +536,43 @@ public static function filter_price_html( $price, $product ) {
 
 		return $price;
 	}
+
+	/**
+	 * Override the WooCommerce "Add to Cart" button text with "Sign Up Now".
+	 *
+	 * @since 1.1.1
+	 */
+	public static function add_to_cart_text( $button_text ) {
+		global $product;
+
+		$product_type = $product->product_type;
+
+		$product_schemes = get_post_meta( $product->id, '_wcsatt_schemes', true );
+
+		$force_subscription = get_post_meta( $product->id, '_wcsatt_force_subscription', true );
+
+		if ( in_array( $product_type, WCS_ATT()->get_supported_product_types() ) && $product_schemes ) {
+
+			if ( 'yes' == $force_subscription ) {
+
+				if ( $product->variation_id > 0 ) {
+
+					$button_text = __( 'Sign Up this Variation', WCS_ATT::TEXT_DOMAIN );
```

Comment:
This button text feels a little too custom to me to include in core. I'd probably instead always use the value of `WC_Subscriptions_Admin::$option_prefix . '_add_to_cart_button_text'` and if the store owner wants custom text for variations, they can use the `'wcsatt_add_to_cart_text'` filter to set that.